src/scripts/run-task.sh

```diff
@@ -68,11 +69,17 @@ if [ -n "$ECS_PARAM_LAUNCH_TYPE" ]; then
     fi
 fi
 
+if [ -n "${ECS_PARAM_PROFILE_NAME}" ]; then
+    set -- "$@" --profile "${ECS_PARAM_PROFILE_NAME}"
+fi
+
 echo "Setting --count"
 set -- "$@" --count "$ECS_PARAM_COUNT"
 echo "Setting --task-definition"
 set -- "$@" --task-definition "$ECS_PARAM_TASK_DEF"
 echo "Setting --cluster"
 set -- "$@" --cluster "$ECS_PARAM_CLUSTER_NAME"
 
+echo "$@" >> test.txt
```

Comment:
This appears to be for debugging

Reply:
Good catch on with the debugging. I removed the `echo "$@" >> test.txt`

> Is any testing required to validate the profile parameter is working?

The `AWS CLI` injects `default` with the `--profile` flag anyways if there isn't a profile specified by the user.  I've added a test for that anyways just to be sure.